transform/mattermost-analytics/models/intermediate/product/nps/_nps__models.yml

```diff
@@ -0,0 +1,53 @@
+version: 2
+
+models:
+
+  - name: int_nps_feedback
+    description: 
+      NPS feedback combined from Staging tables from mattermost_nps & mm_plugin_prod source.
+    
+    columns:
+      - name: server_id
+        description: The ID of the server the event originated from.
+      - name: user_id
+        description: The ID of the user that sent the event.\
+      - name: license_id
+        description: The license id.
+      - name: event_date
+        description: The date the event was sent.
+      - name: timestamp
+        description: Timestamp registered by Rudderstack when the event occurred refactored to clients clock skew.
+      - name: server_version
+        description: The Server's version.
```

Comment:
nit: Server->server to be consustent across.

Reply:
Updated.